test/unit/auth/token-verifier.spec.ts

```diff
@@ -593,110 +490,5 @@ describe('FirebaseTokenVerifier', () => {
       await tokenVerifier.verifyJWT(idTokenNoHeader)
         .should.eventually.be.rejectedWith('Firebase ID token has no "kid" claim.');
     });
```

Comment:
Ideally we need some tests to ensure that:

1. KEY_FETCH_ERROR is handled gracefully, and
2. AppOptions.httpAgent is passed to the verifier.